src/scripts/main.js

```diff
@@ -358,3 +358,24 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+// step 1 . 1
+for (const person of people) {
+  // find table.tbody
+  const table = document.querySelector('.dashboard').firstElementChild;
+  // create tr with 6 cells and inner text for them
+  const row = document.createElement('tr');
+  const personsName = (document.textContent = person.name);
+  const gender = (document.textContent = person.sex);
+  const born = (document.textContent = person.born);
+  const died = (document.textContent = person.died);
+  const age = (document.textContent = died - born);
+  const century = (document.textContent = Math.ceil(died / 100));
```

Comment:
The assignment `document.textContent = Math.ceil(died / 100)` is incorrect. Calculate the century and set it as the text content of a new cell element.